shared-libs/search/src/search.js

```diff
@@ -8,6 +8,10 @@ var _ = require('underscore'),
 module.exports = function(Promise, DB) {
   // Get the subset of rows, in appropriate order, according to options.
   var getPageRows = function(type, rows, options) {
+    if (rows.length < options.skip) {
+      return [];
```

Comment:
I don't get how this works, for me it needs an explanatory comment.

Reply:
Do you think the comment I added is clear enough?